_episodes/02-starting-with-data.md

```diff
@@ -18,7 +18,10 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable you to add functions to Python. Pandas is a popular library for working with data."
```

Comment:
I suggest splitting this into two:
1. Python libraries extend the functionality of Python.
2.

Reply:
...Classic mistake (hitting Return before completing the message)...
2. Pandas is a popular Python library for ....

Reply:
```suggestion
    - "Libraries enable you to add functions to Python." 
    - "Pandas is a popular library for working with data."
```